ols/utils/query_filter.py

```diff
@@ -14,17 +14,17 @@
 # TODO: OLS-380 Config object mirrors configuration
 
 
-class QueryFilter:
+class QueryFilters:
```

Comment:
There was a conflict with `QueryFilter` config model. The new name seems to be more proper anyway.

I've also put `query_filters` as mandatory input instead of using that from the config (it is initialized in the `AppConfig`).